processor/k8sattributesprocessor/README.md

```diff
@@ -196,6 +196,21 @@ the processor associates the received trace to the pod, based on the connection
 }
 ```
 
+By default, the processor will be ready as soon as it starts, even no metadata has been fetched yet.
+If data is sent to this processor before the metadata is synced, there will be no metadata to enrich the data with.
+
+To wait for the metadata to be synced before the processor is ready, set the `wait_for_metadata` option to `true`.
+Then the processor will not be ready until the metadata has been synced.
```

Comment:
Does this mean that the Collector is blocked from getting started until the processor becomes ready?
If that's the case, maybe this should be mentioned explicitly here.

Reply:
```suggestion
Then the processor will not be ready until the metadata is fully synced. As a result, the start-up of the Collector tool will be blocked. If the metadata cannot be synced, the Collector tool will ultimately fail to start.
```

Reply:
Yes. I'll add it.